packages/core/src/common/classes.ts

```diff
@@ -191,8 +191,12 @@ export const SKELETON = "pt-skeleton";
 export const SLIDER = "pt-slider";
 export const SLIDER_HANDLE = "pt-slider-handle";
 export const SLIDER_LABEL = "pt-slider-label";
+export const SLIDER_TRACK = "pt-slider-track";
 export const SLIDER_PROGRESS = "pt-slider-progress";
 export const RANGE_SLIDER = "pt-range-slider";
+export const MULTI_RANGE_SLIDER = "pt-multi-range-slider";
+export const LOWER = "pt-lower";
```

Comment:
what about `start` and `end`? we use this vocabulary in other places already (more generic), and it seems to make sense as well here. In addition, `lower` and `upper` imply a vertical axis, whereas sliders can be either horizontal or vertical

Reply:
Done. Also changed "above" and "below" to "after" and "before"